pit30m/data/log_reader.py

```diff
@@ -376,7 +375,8 @@ def utm_poses_dense(self) -> Tuple[np.ndarray, np.ndarray]:
         submaps = [UUID(bytes=submap_uuid_bytes.ljust(16, b"\x00")) for submap_uuid_bytes in mrp["submap_id"]]
 
         try:
-            xyzs = self._map.to_utm(xyzs, submaps)
+            # XXX: check with Julieta, is it OK to keep this strict?
+            xyzs = self._map.to_utm(xyzs, submaps, strict=False)
```

Comment:
I think it's ok to keep this as non-strict for now. Thanks for checking!

Reply:
Awesome, done and updated the doc to make it more clear what the returns mean.